.github/workflows/p.yml

```diff
@@ -0,0 +1,27 @@
+name: Compile and check P models
+
+on:
+  push:
+    branches:
+      - 'dev'
+    paths:
+      - 'tests/models/**'
+    tags-ignore:
+      - '**'
+  pull_request:
+    paths:
+      - 'tests/models/**'
+
+jobs:
+  cpp:
+    name: Compile and check P models
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+    - name: Compile
+      working-directory: tests/models
+      run: ../../tools/p/p compile
+    - name: Check
+      working-directory: tests/models
+      run: ../../tools/p/p check -tc tcMultiPartitionProduce -t 10
```

Comment:
The expectation is to add an additional step for each test case in the future? Can we love this to a script/makefile so it’s easier to read and easier to run locally the same test?

Reply:
yeh, as soon as there is more than one test